core/settings.py

```diff
@@ -285,7 +285,13 @@
 PELICAN_FRONTEND_URL = os.getenv("PELICAN_FRONTEND_URL")
 
 # WARNING: If you change the production default, update `Dockerfile_django` and `docker-compose.yaml` to match.
-EXPORTER_DIR = os.getenv("EXPORTER_DIR", "/data/exporter" if production else BASE_DIR / "data" / "exporter")
+EXPORTER_MOUNT_DESTINATION = os.getenv(
+    "EXPORTER_MOUNT_DESTINATION", "/data/exporter" if production else BASE_DIR / "data" / "exporter"
+)
+# WARNING: The production default needs to sync with Spoonbill's configuration.
+EXPORTER_MOUNT_SOURCE = os.getenv(
+    "EXPORTER_MOUNT_DESTINATION", "/data/storage/exporter_dumps" if production else BASE_DIR / "data" / "exporter"
```

Comment:
Typo. Also, we can leave the default to be the same as the destination – any deployment will override these.

```suggestion
    "EXPORTER_MOUNT_SOURCE", EXPORTER_MOUNT_DESTINATION
```